src/planscape/rscripts/forsys.R

```diff
@@ -0,0 +1,95 @@
+library("DBI")
```

Comment:
I recommend adding the instructions (posted in the PR description) directly in here since people may find it hard to find the PR for instructions.